### ISSUE TITLE
Find out how the tiles/what elemnts on the tiles in Kat-alogus should be clickable for the best UX

### ISSUE THREAD

**Issue by TwistMeister** [minvws/nl-kat-rocky#47]
Currently the tiles aren't clickable, but the link "see details" is.

It's probably more intuitive to have the tiles be clickable as well as the "see details" link and the "publisher" name. But this should be thought trough carefully, to ensure the best possible UX.

**Comment by NoemiOverdijk** [comment]
I agree, it's definitely more user friendly to make most of the tile clickable/the entire tile clickable. My suggestion would be: 

<img width="721" alt="KAT _ boefjecard" src="https://user-images.githubusercontent.com/118826623/210358492-0b5c2752-b937-424a-ac4b-a7ce4289f18f.png">
